content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,329 @@
 
-**Note**: A `COUNT` is different from the `RATE` metric type, which represents this count of events normalized _per second_ given the defined time interval.
+**Note**: A COUNT is different from the RATE metric type, which represents this count of events normalized per second given the defined time interval.
 
-For example, suppose the `number.of.requests.count` metric is reported every 30 seconds to Datadog with the `COUNT` type on `server:web_1`.
+{{% /tab %}}
+{{% tab "RATE" %}}
 
-Each value/data point for this metric submitted as a `COUNT` represents the number of requests—the "count" of requests—received during the 30 second flush interval. The metric then reports the following values:
+The RATE metric submission type represents the total number of events occurrences per second in one time interval. A RATE can be used to track how often something is happening—like the frequency of connections made to a database or the flow of requests made to an endpoint.
 
-* `10` for the first 30 seconds
-* `20` for the second interval of 30 seconds
-* `null` for the last interval of 30 seconds
+**Note**: A RATE is different from the COUNT metric submission type, which represents the total number of events in the given time interval.
 
-**Note**: for a `COUNT` metric, when a `0` value is submitted, `null` is stored within Datadog.
+{{% /tab %}}
+{{% tab "GAUGE" %}}
 
-When graphed, this `COUNT` metric looks like the following:
+The GAUGE metric submission type represents a snapshot of events in one time interval. This representative snapshot value is the last value submitted to the Agent during a time interval. A GAUGE can be used to take a measure of something reporting continuously—like the available disk space or memory used.
 
-{{< img src="developers/metrics/types/count_metric.png" alt="Count Metric" >}}
+{{% /tab %}}
+{{% tab "HISTOGRAM" %}}
 
-Note: StatsD counts show a decimal value within Datadog, since they are normalized over the flush interval to report units per second.
+The HISTOGRAM metric submission type represents the statistical distribution of a set of values calculated Agent-side in one time interval. Datadog’s HISTOGRAM metric type is an extension of the StatsD timing metric type: the Agent aggregates the values that are sent in a defined time interval and produces different metrics which represent the set of values.
 
-Discover how to submit count metrics:
+If you send `X` values for a HISTOGRAM metric `<METRIC_NAME>` in a given time interval, the following metrics are produced by the Agent by default:
 
-* [With a custom Agent Check][1]
-* [With DogStatsD][2]
-* [With the Datadog API][3]
+| Aggregation                  | Description                                                                                                                                                  | Datadog In-app Type |
```

Comment:
| New Metric Name                  | Description                                                                                                                                                  | Datadog In-App Type |